toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Authenticator.kt

```diff
@@ -82,43 +77,37 @@ public fun Authenticator(
  * For alternate behavior, see the [Authenticator] component.
  *
  * @param authenticatorState the object that holds the state to handle authentication challenges.
- * @param modifier the [Modifier] to be applied to this DialogAuthenticator.
  * @param onPendingOAuthUserSignIn if not null, this will be called when an OAuth challenge is pending
  * and the browser should be launched. Use this if you wish to handle OAuth challenges from your own
  * activity rather than using the [OAuthUserSignInActivity].
  * @see Authenticator
  * @since 200.2.0
  */
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 public fun DialogAuthenticator(
     authenticatorState: AuthenticatorState,
-    modifier: Modifier = Modifier,
     onPendingOAuthUserSignIn: ((OAuthUserSignIn) -> Unit)? = null,
 ) {
-    val showDialog = authenticatorState.isDisplayed.collectAsStateWithLifecycle(initialValue = false).value
+    val showDialog =
+        authenticatorState.isDisplayed.collectAsStateWithLifecycle(initialValue = false).value
     if (showDialog) {
         Surface {
             AuthenticatorDelegate(
                 authenticatorState = authenticatorState,
-                modifier = modifier,
-                onPendingOAuthUserSignIn,
+                //modifier = modifier.clip(MaterialTheme.shapes.extraLarge),
+                onPendingOAuthUserSignIn = onPendingOAuthUserSignIn,
```

Comment:
The modifier needs to be passed to the `AuthenticatorDelegate`
```suggestion
                modifier = modifier,
                onPendingOAuthUserSignIn = onPendingOAuthUserSignIn,
```